vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import pandas
+
+
+# Function to create model
+def emergent_object_model(path_to_weights):
```

Comment:
Instead of a comment, should be using a docstring: https://missourimrr.github.io/docs/python/documenting-code/docstrings/#docstrings